yarn.lock

```diff
@@ -20,7 +20,7 @@
     "@jridgewell/gen-mapping" "^0.3.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
-"@apollo/client@^3.2.2", "@apollo/client@^3.4.8", "@apollo/client@^3.6.2":
+"@apollo/client@^3.4.8", "@apollo/client@^3.6.2":
```

Comment:
nit: do you know why are we referencing multiple versions?

Reply:
`contentful-core` was using a different version than `nextjs-contentful-renderer`. I've updated the version in contentful-core to match